app/components/audio/song-control.tsx

```
@@ -1,57 +1,154 @@
+              src={"/assets/placeholder.jpg"} 
+              alt="song-playing" 
+              width={64} 
+              height={64}
+              className={twMerge(
+                "transition-opacity duration-200",
+                (isLoading || isEmpty) && "grayscale"
+              )}
+            />
+            {isLoading && (
+              <div className="absolute inset-0 flex items-center justify-center">
+                <div className="w-6 h-6 border-2 border-primary border-t-transparent rounded-full animate-spin"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)